docs/pages/fund-agents-apps/fund-your-app.md

```diff
@@ -0,0 +1,201 @@
+# Fund an app to send messages with XMTP
+
+Use this guide to learn how to use the XMTP Funding Portal to fund an app to send messages with XMTP. You can also use the portal to view usage and current and projected fees.
+
+Behind the scenes, the Funding Portal handles Payer Registry smart contract deposits, XMTP Broadcast Network messaging fee and XMTP Appchain gas payments, and all blockchain interactions.
+
+[Ephemera](https://ephemerahq.com/) currently hosts the XMTP Funding Portal UI as a community service. Stewardship will move to DUNA post-GA. 
+
+The [smart contracts](https://github.com/xmtp/smart-contracts) used by the portal are fully decentralized and non-custodial. Full audits from [Trail of Bits](https://www.trailofbits.com/) and [Octane](https://www.octane.security/) will be published before paid messaging is enforced.
+
+## Understand payer wallets
+
+A payer wallet is what an app or agent's XMTP Gateway Service uses to pay fees.
+
+You can fund an app:
+
+- [Using its payer wallet](#fund-an-app-using-its-payer-wallet)
+- [Using a wallet other than its payer wallet](#fund-an-app-using-a-wallet-other-than-its-payer-wallet)
+
+For example:
+
+- You might want to use your payer wallet to fund your payer wallet.
+- You might want to use a different wallet to fund your payer wallet. In this case, you might want to use a multisig wallet to fund your payer wallet. (A multisig wallet can't be payer wallet because a payer wallet must have a private key.)
+- You might want to use a different wallet to support an app or agent by funding its payer wallet.
+
+## Fund an app using its payer wallet
```

Comment:
Since these two flows are quite long and mutually exclusive, maybe we can display them in a tabbed UI?